testsuite/integration/expansion/src/test/java/org/wildfly/test/integration/microprofile/reactive/RunArtemisAmqpSetupTask.java

```diff
@@ -97,6 +107,9 @@ public void setup(ManagementClient managementClient, String containerId) throws
             } catch (Exception ex) {
                 e.printStackTrace();
             }
+            if (e instanceof AssumptionViolatedException ave) {
+                throw ave;
+            }
```

Comment:
I'll file a WFARQ issue, but it looks like an issue that `tearDown()` is not invoked by WildFly Arquillian itself.

Reply:
Yeah. I need to take a closer look, but it seems in some cases the `tearDown()` would happen, but not in all. I just want to make sure it's consistent.